packages/firestore/src/local/simple_db.ts

```diff
@@ -111,7 +111,7 @@ export class SimpleDb {
         // We are provided a version upgrade transaction from the request, so
         // we wrap that in a SimpleDbTransaction to allow use of our friendlier
         // API for schema migration operations.
-        const txn = new SimpleDbTransaction(request.transaction);
+        const txn = new SimpleDbTransaction(request.transaction!);
```

Comment:
transaction is nullable - https://developer.mozilla.org/en-US/docs/Web/API/IDBRequest/transaction
It will always have a value since we use it within onupgradeneeded callback.